core-services/release-controller/_releases/release-ocp-4.10-ci.json

```diff
@@ -25,6 +25,8 @@
           "ovirt-installer",
           "sig-storage-local-static-provisioner",
           "ironic",
+          "ironic-agent",
+          "ironic-hardware-inventory-recorder",
```

Comment:
Context: we don't want to mirror `ironic-hardware-inventory-recorder` out to the `origin` namespace from `ocp`. It's not being used, and if it were we'd want to promote an OKD-specific version to `origin` anyway.